spec/features/see_reviews_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+feature 'see reviews', %Q{
+  As a user,
+  I want to view the reviews for a beer
+  So that my mind can be influenced
+} do
+  # Acceptance Criteria
+  # * I must be on the details page for a specific beer
+  # * I must only see the reviews for the beer I’m viewing
+  # * I must see the reviews listed in order, most recent first
+
+  scenario 'sucessfully see reviews' do
+    beer_1 = FactoryGirl.create(:beer)
+    review_1 = Factorygirl.create(:review, beer: beer_1)
+    review_2 = Factorygirl.create(:review, beer: beer_1)
+    review_3 = Factorygirl.create(:review)
```

Comment:
Instead of all this, look at FactoryGirl's `create_list` method which lets you do all of this in one swoop.

https://horizon.launchacademy.com/lessons/factory-girl
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md